src/main.rs

```diff
@@ -34,6 +34,12 @@ fn main() {
             0.0
         };
 
+        // empty file that is correctly gzipped would otherwise result in minimum_read_length being
```

Comment:
Would just checking the `total` be more robust?

Reply:
i guess it is the same thing (both only happen if `parser.each(|record|` is never entered), but I agree that comparing to zero than some MAX value is more concrete.